packages/components/src/confirm-dialog/test/index.js

```diff
@@ -0,0 +1,284 @@
+// TBD
```

Comment:
This should be cleaned up before merging

Reply:
👍🏻